packages/typescript/package.json

```diff
@@ -1,14 +1,21 @@
 {
   "name": "@frontside/typescript",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "TypeScript Development Tools at Frontside",
   "author": "Frontside Engineering <engineering@frontside.com>",
   "dependencies": {
-    "@frontside/tsconfig": "1.0.0",
-    "eslint": "4.18.2",
-    "typescript": "^3.9.7"
+    "@frontside/eslint-config": "1.2.0",
+    "@frontside/tsconfig": "1.1.0",
+    "@typescript-eslint/eslint-plugin": "^3.10.1",
+    "@typescript-eslint/parser": "^3.10.1",
+    "eslint": "7.7.0",
+    "ts-node": "^9.0.0",
```

Comment:
Until we can confirm that `ts-node` is executable in a monorepo, should we put this off for another PR?

Reply:
I'm actually not a fan of ts-node getting resolved this way.  I've had problems with trying to do this in the past for other things so happy to remove.

For me, the important things are

- upgrade typescript to 4.0
- esModuleInterop: true is in the base config

Reply:
removed ts-node